R/context.R

```diff
@@ -21,3 +21,26 @@ context <- function(desc) {
 end_context <- function() {
   get_reporter()$.end_context()
 }
+
+
+#' @describeIn context Simplify a test filename to a context name.
```

Comment:
I think this could just be it's own documentation file with `@keyword internal`.